0015 Community Membership Process.md

```diff
@@ -0,0 +1,83 @@
+# Process for Defining Community Member Roles  
+
+* Status: proposed
+
+* Input: Welcome from anyone!
+* Deciders:  * Ezra and anyone with merge rights on any repo.
+
+* Date: {2022/07/24 last updated} <!-- optional -->
+
+## Context and Problem Statement
+
+Leadership and Team Structure has been more or less ad hoc. There is a need for a community-driven process to:
+1. Define and recognize needed positions
+2. Formalize who is holding responsibility for a specific position.
+3. Provide new opportunities
+4. Clearly define teams
+5. Replace inactive team members with positions
+
+## Decision Drivers <!-- optional -->
+
+* Functionality - will the new process work and gain buy-in from the community.
+* Scalability - will the process allow for a constant influx of new talent
+* Maintainability - will the process be easy to maintain or self-maintaining
+
+## Considered Options
+
+* Option #1 - Direct new-comers and team members to edit a GitHub team page such as this example: https://github.com/Greenstand/START-Greenstand-overview/blob/master/TEAM.md
+
+This page can function as:
+1. A listing for available positions
+2. A listing of leadership roles
+3. Identifying team members
+4. Place for leads to list and contributors to accept micro roles
+4. A first pull request for novice developers
```

Comment:
I think you may experience a lot of chaos in terms of managing these pull requests, as well as individuals submitting PRs but then never contributing, making for a messy display of team structure and necessitating a consistent and organized cleanup process.

Reply:
Maybe we should make some discussion to verify the candidate is competent before merging a valid PR into the repo,

Reply:
@ZavenArra Thanks for your input. 

Managing pull requests would require a "talent" merger, or input from the leader of that team. You are correct that it would certainly require consistent organized cleanup and management - which I see as part of any system we use to perfect a highly functional remote team. 

@dadiorchen I think clearly defining who can merge these would requests to be part of a team is critical.

As for people never contributing, we could just leave the pull requests open until someone does something. Then merge them into a a team.